### PR TITLE
Adds single lookup support

### PR DESCRIPTION
**Pull request by fayzanx**
- ⚠**Breaking Change:** `ipinfo.lookup` would now accept single IP address instead of an array. Batch lookup is moved to separate method `ipinfo.lookup_batch`
- Would need to create an issue to write unit tests for single lookup @rm-Umar

### CODE REVIEW

**Review by rm-Umar — APPROVED** [review]
@fayzanx I am not good with rust so looks good to me. I think we should add the unit test in the same PR to be sure everything is working as expected. @UmanShahzad if you are good at rust can you have a quick look at it?

File: src/ipinfo.rs [hunk 1]
```diff
@@ -235,6 +224,77 @@ impl IpInfo {
...
+    /// use ipinfo::IpInfo;
+    ///
+    /// let mut ipinfo = IpInfo::new(Default::default()).expect("should construct");
+    /// let res = ipinfo.lookup("8.8.8.8").expect("should run");
+    /// ```
+    pub fn lookup(
+        &mut self,
+        ip: &str,
+    ) -> Result<IpDetails, IpError> {
+
```
Comment (UmanShahzad) [inline src/ipinfo.rs:241]:
```suggestion
```

**Review by UmanShahzad — APPROVED** [review]
LGTM

**Comment by UmanShahzad** [comment]
@rm-Umar can you make a major version change and upload it?

**Pull request merged**